tests/tools/importexport/migration/conftest.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""Module with tests for export archive migrations."""
+import tarfile
+import zipfile
+
+import pytest
+
+from aiida.tools.importexport.archive import safe_extract_tar, safe_extract_zip, CacheFolder
+from aiida.tools.importexport.archive.migrations.utils import verify_metadata_version
+from tests.utils.archives import get_archive_file
+
+
+@pytest.fixture()
+def core_archive():
```

Comment:
the core archive or ones that point to files in the aiida-core repository, as opoosed to the external ones, which point to the `aiida-export-migration-tests` package.
Although I did point out that having the archives in a separate repo may not really be necessary anymore: https://github.com/aiidateam/aiida-export-migration-tests/issues/13

Reply:
it was just the easiest way really, to minimise the changes require during the conversion pytest. I don't think its too bad, if they are tightly scoped (i.e. the conftest only applies to this single folder),
plus its a lot easier now that vs code's intellisense recognises pytest fixtures, so that they exactly the same as if you had imported them 😄

Reply:
> Although I did point out that having the archives in a separate repo may not really be necessary anymore: [aiidateam/aiida-export-migration-tests#13](https://github.com/aiidateam/aiida-export-migration-tests/issues/13)

Right, you are welcome to change this here (not sure when someone will touch this for the next time.

By the way, there is also the git-lfs for storing larger binary files so it wouldn't even clog up repository size; I think git now automatically suggests it when trying to store larger files.

Reply:
interesting thanks I will check that out